.github/workflows/update-tag.yml

```diff
@@ -0,0 +1,20 @@
+name: Upgrade v1 tag
+
+on:
+  push:
+    branches: [main]
+
+jobs:
+  build:
+    runs-on: ubuntu-22.04
+    env:
+      GITHUB_TOKEN: ${{ secrets.RELENG_GH_TOKEN }}
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          token: ${{ secrets.RELENG_GH_TOKEN }}
+      - name: Upgrade tag
```

Comment:
noob question: is there a way to make this depend on the integration test success? We will still have the possibility to release manually but this would prevent us from releasing something obviously broken, e.g. because of a merge conflict.

Reply:
Yep! I've pushed a commit that I believe should do this: https://github.com/lacework/code-security-action/pull/65/commits/f74a0860b209b27e61c0fe84fa13fa55205a392c

Reply:
Nice!